master/internal/experiment/bulk_action.go

```diff
@@ -185,6 +192,7 @@ func editableExperimentIds(ctx context.Context, inputExpIDs []int32,
 		Model(&expIDs).
 		Column("e.id").
 		Join("JOIN projects p ON e.project_id = p.id").
+		Where("NOT e.archived").
```

Comment:
what is this for? i'm not sure this is generally true, that you can't edit archived (e.g., needing to unarchive something to delete it is weird).

Reply:
we call editableExperimentIds from Activate, Cancel, Kill, and Pause; so Delete would be a separate situation